Makefile

```diff
@@ -28,7 +28,7 @@ lint:
 
 .PHONY: mypy
 mypy:
-	mypy pydantic_extra_types
+	@mypy --install-types --non-interactive pydantic_extra_types
```

Comment:
Can you add Types to Linting requirements rather than changing mypy to use `--install-types`? 🤔

Reply:
i tried and does not work see last commit

Reply:
@yezz123 how do i resolve this ? you can try yourself

Reply:
@yezz123  can i keep --install-types since installing linting requirements does not work ?

Reply:
id did the trick